app/i18n/en-us/admin.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+return array(
+	'auth' => array(
+		'allow_anonymous' => 'Allow anonymous reading of the default user\'s articles (%s)',
+		'allow_anonymous_refresh' => 'Allow anonymous refresh of the articles',
+		'api_enabled' => 'Allow <abbr>API</abbr> access <small>(required for mobile apps)</small>',
+		'form' => 'Web form (traditional, requires JavaScript)',
+		'http' => 'HTTP (for advanced users with HTTPS)',
+		'none' => 'None (dangerous)',
+		'title' => 'Authentication',
+		'title_reset' => 'Authentication reset',
+		'token' => 'Authentication token',
+		'token_help' => 'Allows access to RSS output of the default user without authentication:',
+		'type' => 'Authentication method',
+		'unsafe_autologin' => 'Allow unsafe automatic login using the format:	-> todo',
```

Comment:
Is that weird todo supposed to be in here?

Reply:
Honestly, I don't know where that came from or how it snuck in here.  gone